reference/5.1/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -191,10 +191,41 @@ Using is a special scope modifier that identifies a local variable in a remote
 command. Without a modifier, PowerShell expects variables in remote commands
 to be defined in the remote session.
 
-The Using scope modifier is introduced in PowerShell 3.0.
+The `Using` scope modifier is introduced in PowerShell 3.0.
+
+r any script or command that executes out of session, you need the `Using`
+scope modifier to embed variable values from the calling session scope, so that
+out of session code can access them. The `Using` scope modifier is supported in
+the following contexts:
+
+- Remotely executed commands, started with `Invoke-Command` using the
+  **ComputerName** or **Session** parameter (remote session)
+- Background jobs, started with `Start-Job` (out-of-process session)
+- Thread jobs, started via `Start-ThreadJob` (separate runspace)
+
+Depending on the context, embedded variables are copies values or references to
```

Comment:
Something is not right here: "..are copies values or.." 
Maybe: "copied values"

Reply:
Good point, but here too I suggest using the same wording as [proposed above](https://github.com/MicrosoftDocs/PowerShell-Docs/pull/5596/files/c31496fddf5130a8c218de2f1da3cac9a2198d58#r392551009), across all similar passages.

Reply:
🤦‍♂ thanks for stopping me right there. Still gets me, all the different copies.

Reply:
@Jawz84 :) - yes, it gets confusing.

Reply:
Fixed